### DISCUSSION TITLE
Copilot Version Error VSCode

### DISCUSSION THREAD

**Discussion by Gnkalk** [community/community#7249] [Projects and Issues] [answered]
My vscode is in the latest update and its version is more up to date than 1.57 but copilot errors and doesn't work.

![photo_2021-11-04_08-34-54](https://user-images.githubusercontent.com/74614163/140260874-31aa43be-9a82-419b-94d6-eb940681fdc6.jpg)

![photo_2021-11-04_08-34-48](https://user-images.githubusercontent.com/74614163/140260877-e9087321-7526-4e21-8a4a-c6ff835329c6.jpg)

**Comment by davecheney** [comment, marked as answer]
@Gnkalk it is likely that you have installed OSS Code which is commonly packaged by linux distributions. Please install VSCode from directly from https://code.visualstudio.com/docs/setup/linux

**Reply by Gnkalk** [reply]
Oh, Thank's.
